spec/unit/util/run_mode_spec.rb

```diff
@@ -2,6 +2,13 @@
 require 'spec_helper'
 
 describe Puppet::Util::RunMode do
+
+  # Discriminator for tests that attempts to unset HOME since that, for reasons currently unknown,
```

Comment:
Were there particular platforms you saw having a problem with unsetting `HOME` @thallgren ?